test/LEAF_Nexus_Tests/tests/api/controllers/PositionControllerTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types = 1);
+
+use LEAFTest\LEAFClient;
+
+/**
+ * Tests the LEAF_Nexus/api/?a=group API
+ */
+class PositionControllerTest extends DatabaseTest
+{
+    private static $client = null;
+
+    protected function setUp()
+    {
+        $this->resetDatabase();
+        self::$client = LEAFClient::createNexusClient();
+    }
+
+    /**
+     * Tests the `position/<id>/supervisor` endpoint
+     */
+    public function testSetSupervisor() : void
+    {
+        $results = self::$client->get('position/3/supervisor');
+        $this->assertEquals("Medical Center Director", $results[0]["positionTitle"]);
+        //initial value
```

Comment:
Comments that describe what is happening in code should precede the statements they describe.